### PR TITLE
REF: SquareEnumaratedWords to TS

### PR DESCRIPTION
**Pull request by Danswar**
Component to Typescript + removing inline styles

### CODE REVIEW

File: components/SquareEnumeratedWords.tsx [hunk 1]
```diff
@@ -1,16 +1,25 @@
...
+export const SquareEnumeratedWordsContentAlign: Record<string, ContentAlignType> = Object.freeze({
+  left: 'flex-start',
+  center: 'center',
+  right: 'flex-end',
+});
+
+interface SquareEnumeratedWordsProps {
+  entries?: string[];
+  appendNumber?: boolean;
+  contentAlign?: ContentAlignType;
```
Comment (limpbrains) [inline components/SquareEnumeratedWords.tsx:15]: I'm looking at how this component is used in ﻿viewEditMultisigCosigners.tsx. ﻿`entries` and ﻿`contentAlign` props are always defined; ﻿`appendNumber` is always false. 

In my opinion, it's worth removing default values, marking all properties as required

**Review by limpbrains — CHANGES_REQUESTED** [review]

**Comment by Danswar** [comment]
@limpbrains I believe this is ready for a second look

**Review by limpbrains — APPROVED** [review]

**Comment by limpbrains** [comment]
@marcosrdz build is failing, can you please take a look?

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by Overtorment** [comment]
builds failing probably due to env secrets not injected on CI, because of out sidecontributor

**Pull request merged**